cmd/build.go

```diff
@@ -44,28 +45,27 @@ var buildCmd = &cobra.Command{
 		}
 
 		syscalls := make([]string, 0)
+		var syscallList = make(map[string]int)
 		for _, fileObj := range files {
-			//fmt.Println("[" + fileObj.Name() + "]")
-
 			file, err := os.Open(inputDirectory + "/" + fileObj.Name())
 			if err != nil {
 				return fmt.Errorf("error opening file %s: %w", file.Name(), err)
```

Comment:
This code will panic

os.Open uses os.OpenFile

OpenFile uses `return nil, err` on error


https://cs.opensource.google/go/go/+/refs/tags/go1.22.5:src/os/file.go;l=372

Usually the file path is stored in a variable, then the error message uses the variable

Reply:
Yea can't use file.Name() if os.Open has failed :+1:

Reply:
right, thanks!